blueprints/ember-redux/files/__root__/reducers/index.js

```diff
@@ -0,0 +1,18 @@
+import { combineReducers } from 'redux';
+
+/**
+ * This is the master reducer, it partitions the jobs amounts
+ * one or more other reducers which take on responsibility for
+ * a discrete part of the state tree.
+ *
+ * Use the "ember generate reducer" command to create additional
+ * reducers.
+ */
+
+// import foo from './foo';
+// import bar from './bar';
+
+export default combineReducers({
```

Comment:
Here's a screen shot from a recent presentation of his:
![2016-09-15_19-20-52](https://cloud.githubusercontent.com/assets/980757/18562177/8dca482c-7b7a-11e6-86fc-846deb2a6706.png)


And with my annotations:

![2016-09-15_19-20-52](https://cloud.githubusercontent.com/assets/980757/18562122/62db4b20-7b7a-11e6-8174-db5c7bf3961e.png)